packages/errors/package.json

```diff
@@ -23,14 +23,14 @@
   },
   "scripts": {
     "build": "exit 0",
-    "clean": "git clean -f '*.d.ts*'",
+    "clean": "git clean -fX \"*.d.ts*\" \"*.d.cts*\" \"*.d.mts*\" \"*.tsbuildinfo\"",
     "lint": "yarn lint:types && yarn lint:eslint",
     "lint-check": "yarn lint",
     "lint-fix": "yarn lint:eslint --fix && yarn lint:types",
     "lint:eslint": "eslint '**/*.js'",
     "lint:types": "tsc",
     "prepack": "tsc --build tsconfig.build.json",
-    "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
+    "postpack": "yarn clean",
```

Comment:
This is probably fine, but I’m reminded that we should avoid using `yarn` in scripts that might be invoked under `npm install` outside our workspace. This is not among them, but it might be a good habit to start using `npm` (or eventually `node --run clean` here).